pkg/k8shandler/elasticsearch.go

```diff
@@ -475,8 +475,9 @@ func (elasticsearchRequest *ElasticsearchRequest) AddAliasForOldIndices() bool {
 		}
 
 		if payload.ResponseBody[index] != nil {
-			indexBody := payload.ResponseBody[index].(map[string]interface{})
-			if indexBody["aliases"] != nil {
+			indexRawBody, okIndex := payload.ResponseBody[index]
```

Comment:
nit but this seems out of scope for this bug fix... can we open up another bz to track this work?

Reply:
Prove me wrong, but i see regurarly the operator panic'ing on e2e.

Reply:
I'll remove and open a separate changeset.